### PR TITLE
Add best practices for audio setup

### PR DESCRIPTION
**Pull request by jeppester**
If you have further advice, please add it :-)

### CODE REVIEW

File: best-practices/audio-setup.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- Open the sound settings on your computer and check if your microphone volume is too high. If it's nearly maxed out, try lowering the volume a bit. Maxing out the microphone can causes severe quality degregation.
+
+### My colleauges tell me that I'm breathing heavily
+
+- Place the microphone "next to" your mouth instead of in front of it
+- Move the microphone further away from your mouth while keeping it pointed towards it.
+
+### My ModMic is picking up a LOT of background noise
+
+- If it has a omni-/uni direction switch, swith it to uni. The correct icon is a circle with a "dent", as opposed to the full circle icon which symbolizes omni-directional mode.
```
Comment (bohme) [inline best-practices/audio-setup.md:16]:
```suggestion
- If it has an omni-/uni direction switch, swith it to uni. The correct icon is a circle with a "dent" (Cardioid), as opposed to the full circle icon which symbolizes omni-directional mode.
```

File: best-practices/audio-setup.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Audio setup
+
+## Known issues and workarounds
+
+### My colleagues tell me that I sound like a pilot
+
+- Open the sound settings on your computer and check if your microphone volume is too high. If it's nearly maxed out, try lowering the volume a bit. Maxing out the microphone can causes severe quality degregation.
```
Comment (mathias-abtion) [inline best-practices/audio-setup.md:7]: This is just minor nitpicking but "degregation " should be "degradation". :-)

**Review by bohme — APPROVED** [review]
Very nice work :3

File: best-practices/audio-setup.md (hunk 2 above)
Comment (jeppester) [reply best-practices/audio-setup.md:7]: I knew this was wrong, I just couldn't point out the exact problem :D

Comment (jeppester) [inline best-practices/audio-setup.md:7]:
```suggestion
- Open the sound settings on your computer and check if your microphone volume is too high. If it's nearly maxed out, try lowering the volume a bit. Maxing out the microphone volume can severely degrade the quality.
```

**Review by mathias-abtion — APPROVED** [review]

**Review by hcarreras — APPROVED** [review]
👏

**Pull request merged**